windows/RNPrint/RNPrint.h

```diff
@@ -0,0 +1,402 @@
+#pragma once
```

Comment:
this file is getting pretty large :) (github wouldn't show me the diff)
Can you break it up into a header and a source?